### PR TITLE
Fixed paths

### PR DESCRIPTION
**Pull request by kayeekayee**
# Description
1) Fix CCPP_SUITE to mynn.
2) Fix all STMP and PTMP path to mynn.

# Type of change
- Bug fix (fixes something broken)

# Change characteristics
- Is this a breaking change (a change in existing functionality)? NO
- Does this change require a documentation update? NO

# How has this been tested?
Ran sample run on Hera /scratch1/BMC/gsd-fv3-dev/KaYee.Wong/github/gsl_ufs_rt

# Checklist
- [x] Any dependent changes have been merged and published
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [x] New and existing tests pass with my changes
- [x] I have made corresponding changes to the documentation if necessary

### CODE REVIEW

File: workflow/gsl_template_hera.xml
Comment (jkhender) [inline workflow/gsl_template_hera.xml:1]: EXPDIR and ROTDIR do not need to be changed

File: workflow/gsl_template_jet.xml
Comment (jkhender) [inline workflow/gsl_template_jet.xml:1]: EXPDIR and ROTDIR do not need to be changed

File: workflow/p8_ugwpv1.sh
Comment (jkhender) [inline workflow/p8_ugwpv1.sh:1]: COMROT and EXPDIR do not need to be changed

File: workflow/rocoto/workflow_xml_gsl.py
Comment (jkhender) [inline workflow/rocoto/workflow_xml_gsl.py:1]: EXPDIR and ROTDIR do not need to be changed

File: workflow/rt_mynn_hera.sh
Comment (jkhender) [inline workflow/rt_mynn_hera.sh:1]: COMROT and EXPDIR do not need to be changed

File: workflow/rt_mynn_jet.sh
Comment (jkhender) [inline workflow/rt_mynn_jet.sh:1]: COMROT and EXPDIR do not need to be changed

File: workflow/test_hera.sh
Comment (jkhender) [inline workflow/test_hera.sh:1]: COMROT and EXPDIR do not need to be changed

File: workflow/test_jet.sh
Comment (jkhender) [inline workflow/test_jet.sh:1]: COMROT and EXPDIR do not need to be changed

**Review by jkhender — CHANGES_REQUESTED** [review]
COMROT should be **$GITDIR**/FV3GFSrun
EXPDIR should be **$GITDIR**/FV3GFSwfm

when you run the Python scripts, then the directories **$GITDIR**/FV3GFSwfm/**$PSLOT** and **$GITDIR**/FV3GFSrun/**$PSLOT** will be created.

**Review by jkhender — APPROVED** [review]
Looks good!

**Review by jderrico-noaa — APPROVED** [review]

**Pull request merged**